code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -30,7 +30,7 @@
 	var/mob/living/carbon/C = owner
 	if(!length(C.stomach_contents))
 		if(!silent)
-			to_chat(C, "<span class='warning'>There's nothing in our belly that needs regurgitating.</span>")
+			to_chat(C, "<span class='warning'>There's nothing in your belly that needs regurgitating.</span>")
```

Comment:
Why are you doing this change in tank pr. You have unrelated changes in a lot of places